### PR TITLE
Dark mode: Navs & tabs

### PR DESCRIPTION
**Pull request by hannahiss**
### Description

Navs & tabs in dark mode, by using existing Sass vars :
| Sass var | Previous value | New value |
| :------- | :--------------: | :----------: |
| `$nav-tabs-link-hover-border-color` | `$border-color` | `var(--#{$prefix}highlight-color)`  ⚠️ strange use of this variable |
| `$nav-underline-border-color` | `$gray-500` | `var(--#{$prefix}border-color-translucent)` |
| `$nav-underline-link-active-color` | `var(--#{$prefix}emphasis-color)` | `var(--#{$prefix}hover-color)` |
| `$nav-underline-link-hover-color` | `$accessible-orange` | `var(--#{$prefix}link-hover-color)`  ⚠️ strange use of this variable |
| `$nav-underline-link-active-bg` | `var(--#{$prefix}body-bg)` | `transparent` ❓ do we need to keep these vars ? (only in Boosted) |
| `$nav-underline-link-active-border-color` | `$accessible-orange` | `var(--#{$prefix}link-hover-color)`  ⚠️ strange use of this variable |

⚠️  Added 2 Sass vars and corresponding css vars:
| Sass var | css var | Value |
| :------- | :--------------: | :----------: |
| `$nav-tabs-link-hover-color` | `--bs-nav-tabs-link-hover-color` | `var(--#{$prefix}highlight-color)` |
| `$nav-tabs-link-hover-bg` | `--bs-nav-tabs-link-hover-bg` | `var(--#{$prefix}highlight-bg)` |

New CSS variable

`--bs-tab-content-border-color` in `.nav-tabs`

### Links

- https://deploy-preview-2307--boosted.netlify.app/docs/5.3/components/navs-tabs/
- https://deploy-preview-2307--boosted.netlify.app/docs/5.3/dark-mode/#navs--tabs

### CODE REVIEW

File: site/content/docs/5.3/dark-mode.md [hunk 1]
```diff
@@ -2029,3 +2034,401 @@ sitemap_exclude: true
...
+        <li><hr class="dropdown-divider"></li>
+        <li><a class="dropdown-item" href="#">Separated link</a></li>
+      </ul>
+    </li>
+  </ul>
+
+
+  <ul role="tablist" aria-owns="nav-tab1 nav-tab2 nav-tab3 nav-tab4" class="nav nav-tabs mt-2" id="nav-tab-with-nested-tabs-2">
+    <li class="nav-item" role="presentation">
+      <a class="nav-link active" aria-current="page" id="nav-tab1-2" href="#tab1-content" data-bs-toggle="tab" data-bs-target="#tab1-content" role="tab" aria-controls="tab1-content" aria-selected="true">Tab 1</a>
```
Comment (louismaximepiton) [inline site/content/docs/5.3/dark-mode.md:2173]:
```suggestion
      <a class="nav-link active" aria-current="page" id="nav-tab1-2" href="#tab1-content-2" data-bs-toggle="tab" data-bs-target="#tab1-content-2" role="tab" aria-controls="tab1-content-2" aria-selected="true">Tab 1</a>
```

File: site/content/docs/5.3/dark-mode.md [hunk 2]
```diff
@@ -2029,3 +2034,401 @@ sitemap_exclude: true
...
+    <li class="nav-item" role="presentation">
+      <a class="nav-link disabled" id="nav-tab4-2" data-bs-toggle="tab" data-bs-target="#tab4-content" role="tab" aria-controls="tab4-content" aria-selected="false" aria-disabled="true">Tab 4</a>
+    </li>
+  </ul>
+
+  <div class="tab-content" id="nav-tabs-content-2">
+    <div class="tab-pane-with-nested-tab fade show active" id="tab1-content-2" role="tabpanel" aria-labelledby="nav-tab1">
+      <ul role="tablist" aria-owns="nav-linkA nav-linkB nav-linkC nav-linkD" class="nav nav-underline mt-0">
+        <li class="nav-item" role="presentation">
+          <a class="nav-link active" id="nav-linkA-2" href="#linkA" data-bs-toggle="tab" data-bs-target="#linkA" role="tab" aria-current="page">Link A</a>
```
Comment (louismaximepiton) [inline site/content/docs/5.3/dark-mode.md:2190]:
```suggestion
          <a class="nav-link active" id="nav-linkA-2" href="#linkA-2" data-bs-toggle="tab" data-bs-target="#linkA-2" role="tab" aria-current="page">Link A</a>
```

File: site/content/docs/5.3/dark-mode.md [hunk 3]
```diff
@@ -2029,3 +2034,401 @@ sitemap_exclude: true
...
+    </li>
+  </ul>
+
+
+  <ul role="tablist" aria-owns="nav-tab1 nav-tab2 nav-tab3 nav-tab4" class="nav nav-tabs mt-2" id="nav-tab-with-nested-tabs-2">
+    <li class="nav-item" role="presentation">
+      <a class="nav-link active" aria-current="page" id="nav-tab1-2" href="#tab1-content-2" data-bs-toggle="tab" data-bs-target="#tab1-content-2" role="tab" aria-controls="tab1-content-2" aria-selected="true">Tab 1</a>
+    </li>
+    <li class="nav-item" role="presentation">
+      <a class="nav-link" id="nav-tab2-2" data-bs-toggle="tab" href="#tab2-content" data-bs-target="#tab2-content" role="tab" aria-controls="tab2-content" aria-selected="false">Tab 2</a>
```
Comment (louismaximepiton) [inline site/content/docs/5.3/dark-mode.md:2176]:
```suggestion
      <a class="nav-link" id="nav-tab2-2" data-bs-toggle="tab" href="#tab2-content-2" data-bs-target="#tab2-content-2" role="tab" aria-controls="tab2-content-2" aria-selected="false">Tab 2</a>
```

File: site/content/docs/5.3/dark-mode.md [hunk 4]
```diff
@@ -2029,3 +2034,401 @@ sitemap_exclude: true
...
+  </ul>
+
+  <div class="tab-content" id="nav-tabs-content-2">
+    <div class="tab-pane-with-nested-tab fade show active" id="tab1-content-2" role="tabpanel" aria-labelledby="nav-tab1">
+      <ul role="tablist" aria-owns="nav-linkA nav-linkB nav-linkC nav-linkD" class="nav nav-underline mt-0">
+        <li class="nav-item" role="presentation">
+          <a class="nav-link active" id="nav-linkA-2" href="#linkA-2" data-bs-toggle="tab" data-bs-target="#linkA-2" role="tab" aria-current="page">Link A</a>
+        </li>
+        <li class="nav-item" role="presentation">
+          <a class="nav-link" id="nav-linkB-2" href="#linkB" data-bs-toggle="tab" data-bs-target="#linkB" role="tab">Link B</a>
```
Comment (louismaximepiton) [inline site/content/docs/5.3/dark-mode.md:2193]:
```suggestion
          <a class="nav-link" id="nav-linkB-2" href="#linkB-2" data-bs-toggle="tab" data-bs-target="#linkB-2" role="tab">Link B</a>
```

**Review by louismaximepiton — COMMENTED** [review]
Sorry I wasn't clear, but all ids mentionned here need a little shift. For instance:
- `tab2-content`
- `tab3-content`
- `tab4-content`
- `linkB`
- `linkC`
- `linkD`

File: site/content/docs/5.3/dark-mode.md (hunk 3 above)
Comment (hannahiss) [reply site/content/docs/5.3/dark-mode.md]: sorry...

**Review by louismaximepiton — APPROVED** [review]

**Review by julien-deramond — APPROVED** [review]

**Pull request merged**